pallets/liquidity-pools-gateway/src/lib.rs

```diff
@@ -29,21 +29,18 @@
 use core::fmt::Debug;
 
 use cfg_primitives::{LP_DEFENSIVE_WEIGHT, LP_DEFENSIVE_WEIGHT_REF_TIME};
-use cfg_traits::{
```

Comment:
@cdamian sorry, I've needed to remove some lines from the gateway, but because they are removed lines, I think the git conflicts are easy to solve, by just applying always these changes in these files

Reply:
No problem, as long as there are no big changes to the actual gateway logic, we can manage. Thank you!